definitions/infra-powerdns-recursor/definition.yml

```diff
@@ -0,0 +1,16 @@
+domain: INFRA
+type: POWERDNS_RECURSOR
```

Comment:
The name of the folders should be `domain`-`type`, so `infra-powerdns-recursor` should be `infra-powerdns_recursor`, same for the other entity type.

Reply:
Fixed, thanks!